Installation fails without a terminal

https://github.com/awslabs/git-secrets/blob/80230afa8c8bdeac766a0fece36f95ffaa0be778/git-secrets#L208

There's no particularly good reason to depend on `say` _and_ to require a valid terminal just to print out a green checkmark symbol upon successful installation. :)

```
Run git secrets --install
tput: No value for $TERM and no -T specified
tput: No value for $TERM and no -T specified
/usr/local/bin/git-secrets: line 208: say: command not found
tput: No value for $TERM and no -T specified
tput: No value for $TERM and no -T specified
/usr/local/bin/git-secrets: line 208: say: command not found
tput: No value for $TERM and no -T specified
tput: No value for $TERM and no -T specified
/usr/local/bin/git-secrets: line 208: say: command not found
Error: Process completed with exit code 127.
```

--- a/git-secrets
+++ b/git-secrets
@@ -205,7 +205,8 @@ install_hook() {
   echo "#!/usr/bin/env bash" > "${dest}"
   echo "git secrets --${cmd} -- \"\$@\"" >> "${dest}"
   chmod +x "${dest}"
-  say "$(tput setaf 2)✓$(tput sgr 0) Installed ${hook} hook to ${dest}"
+  [ -t 1 ] && which tput >/dev/null && echo -n "$(tput setaf 2)✓$(tput sgr 0) "
+  echo "Installed ${hook} hook to ${dest}"
 }
 
 install_all_hooks() {

--- a/git-secrets
+++ b/git-secrets
@@ -205,7 +205,7 @@ install_hook() {
   echo "#!/usr/bin/env bash" > "${dest}"
   echo "git secrets --${cmd} -- \"\$@\"" >> "${dest}"
   chmod +x "${dest}"
-  [ -t 1 ] && which tput >/dev/null && echo -n "$(tput setaf 2)✓$(tput sgr 0) "
+  [ -t 1 ] && command -v tput &> /dev/null && echo -n "$(tput setaf 2)✓$(tput sgr 0) "
   echo "Installed ${hook} hook to ${dest}"
 }
 

